readme.md

```diff
@@ -296,6 +382,10 @@ unzip /tmp/data/berlin.zip -d /tmp/data
 
 # download openstreetmap data
 curl -s https://s3.amazonaws.com/metro-extracts.mapzen.com/berlin_germany.osm.pbf > /tmp/data/berlin.osm.pbf
+
+# download tiger data (note data directory will be created if it doesn't exist)
+export PELIAS_CONFIG=./pelias.json
```

Comment:
is this strictly required or just a hint for users who don't want to create the `pelias.json` file in their home directory?